### ISSUE TITLE
[16.0] upgrade_analysis : KeyError: 'module' when generating records due to patched method init_models

### ISSUE THREAD

**Issue by metaminux** [OCA/server-tools#2622]
## Module
upgrade_analysis

## Describe the bug
Using the Generate Records Wizard raises an RPC_ERROR: Odoo Server Error.

```
Traceback (most recent call last):
  File "/opt/odoo/odoo/http.py", line 1583, in _serve_db
    return service_model.retrying(self._serve_ir_http, self.env)
  File "/opt/odoo/odoo/service/model.py", line 134, in retrying
    result = func()
  File "/opt/odoo/odoo/http.py", line 1610, in _serve_ir_http
    response = self.dispatcher.dispatch(rule.endpoint, args)
  File "/opt/odoo/odoo/http.py", line 1807, in dispatch
    result = self.request.registry['ir.http']._dispatch(endpoint)
  File "/opt/odoo/odoo/addons/base/models/ir_http.py", line 154, in _dispatch
    result = endpoint(**request.params)
  File "/opt/odoo/odoo/http.py", line 696, in route_wrapper
    result = endpoint(self, *args, **params_ok)
  File "/opt/odoo/addons/web/controllers/dataset.py", line 46, in call_button
    action = self._call_kw(model, method, args, kwargs)
  File "/opt/odoo/addons/web/controllers/dataset.py", line 33, in _call_kw
    return call_kw(request.env[model], method, args, kwargs)
  File "/opt/odoo/odoo/api.py", line 461, in call_kw
    result = _call_kw_multi(method, model, args, kwargs)
  File "/opt/odoo/odoo/api.py", line 448, in _call_kw_multi
    result = method(recs, *args, **kwargs)
  File "/mnt/data/odoo-addons-dir/upgrade_analysis/wizards/upgrade_generate_record_wizard.py", line 52, in generate
    Registry.new(self.env.cr.dbname, update_module=True)
  File "<decorator-gen-14>", line 2, in new
  File "/opt/odoo/odoo/tools/func.py", line 87, in locked
    return func(inst, *args, **kwargs)
  File "/opt/odoo/odoo/modules/registry.py", line 90, in new
    odoo.modules.load_modules(registry, force_demo, status, update_module)
  File "/opt/odoo/odoo/modules/loading.py", line 506, in load_modules
    registry.init_models(cr, list(models_to_untranslate), {'models_to_check': True})
  File "/mnt/data/odoo-addons-dir/upgrade_analysis/odoo_patch/odoo/modules/registry.py", line 18, in init_models
    module_name = context["module"]
KeyError: 'module'

The above server error caused the following client error:
RPC_ERROR: Odoo Server Error
    RPCError@http://oca-server-tools-16-0-c0064d6939cb.runboat.odoo-community.org/web/assets/441-e8d050c/web.assets_backend.min.js:966:274
    makeErrorFromResponse@http://oca-server-tools-16-0-c0064d6939cb.runboat.odoo-community.org/web/assets/441-e8d050c/web.assets_backend.min.js:970:163
    jsonrpc/promise</<@http://oca-server-tools-16-0-c0064d6939cb.runboat.odoo-community.org/web/assets/441-e8d050c/web.assets_backend.min.js:978:34
```

## To Reproduce
**Affected versions**: 16.0

1. Create a new database
2. Install upgrade_analysis module
3. Click on Settings / Upgrade Analysis / Generate Records Wizard

**Expected behavior**
Records should be generated, the wizard state should be Done.

**Additional context**
Tested on local database with only upgrade_analysis module installed and on runboat
Some records are generated anyway, but I don't know if everything is here.

**Comment by legalsylvain** [comment]
Cc @remytms .

**Comment by metaminux** [comment]
#2624

**Comment by metaminux** [comment]
Fixed by #2624

**Comment by legalsylvain** [comment]
thanks for reporting and for the fix !
